crates/oxc_linter/src/snapshots/no_unused_vars@oxc-classes.snap

```diff
@@ -0,0 +1,28 @@
+---
+source: crates/oxc_linter/src/tester.rs
+---
+  ⚠ eslint(no-unused-vars): Parameter 'a' is declared but never used.
+   ╭─[no_unused_vars.tsx:1:32]
+ 1 │ export class Foo { constructor(a: number) {} }
+   ·                                ────┬────
+   ·                                    ╰── 'a' is declared here
```

Comment:
this is very much a nit, but would be great if this could only report on the `a` (identifier) rather than the whole arg

Reply:
I agree. Spans for symbols currently cover both the binding and the type annotation. I'll make a separate PR updating this. CC: @Dunqing  @overlookmotel

Reply:
Nevermind, this is intentional behavior for the parser. See #2624. Is it worth trying to subtract out type annotations from the reported span?

Reply:
I have looked at `TypeScript`, which doesn't cover the type annotation span.